instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty/v4_1/ChannelFutureListenerInstrumentation.java

```diff
@@ -69,7 +69,7 @@ public ChannelFutureListenerInstrumentation() {
       packageName + ".client.HttpClientResponseTracingHandler",
       packageName + ".client.HttpClientTracingHandler",
       // server helpers
-      packageName + ".server.NettyHttpServerDecorator",
+      packageName + ".server.NettyHttpServerTracer",
```

Comment:
i think all these server helpers can go away

Reply:
Why? From class loading perspective they don't differ from old decorator, do they?

Reply:
A, I see what you mean... Let me try

Reply:
Nope, they cannot. `ChannelFutureListenerInstrumentation` uses `AttributeKeys`, which depends on `HttpServerTracingHandler`